### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Michadelic**
* add travis build image
* add OpenUI5 logo
* simplify readme text and contributing instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
-# Repository for OpenUI5 website
+[![Build Status](https://travis-ci.org/SAP/openui5-website.svg?branch=master)](https://travis-ci.org/SAP/openui5-website)
+![OpenUI5 logo](http://openui5.org/images/OpenUI5_new_big_side.png)
```
Comment (OlMue) [inline README.md]: Good idea

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
-This repository contains the source files for the [openui5.org](openui5.org) website of [OpenUI5](https://github.com/SAP/openui5) framework.
+# openui5-website
 
-We welcome your contributions to our website. If you have a proposal you want to share with us, simply keep the following things in mind.
+Repository for the [openui5.org](openui5.org) website of the [OpenUI5](https://github.com/SAP/openui5) framework.
```
Comment (OlMue) [inline README.md]: I prefer a full sentence, it sounds more natural. Proposal: This the repository for the [openui5.org](openui5.org) website of the [OpenUI5](https://github.com/SAP/openui5) framework.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 
 ## Contributing
 
-You only need to follow the installation instructions below if you want to contribute something.
+Contributions welcome! If you have a proposal you want to share with us, please follow the steps listed below.
```
Comment (OlMue) [inline README.md]: Maybe mix the two versions: Contributions welcome! If you have a proposal you want to share with us, simply follow the steps listed below.

File: README.md (hunk 1 above)
Comment (Michadelic) [reply README.md]: ok, didn't know that, will revert

**Pull request merged**